pynamodb/models.py

```diff
@@ -186,6 +186,12 @@ def __init__(cls, name, bases, attrs):
             if META_CLASS_NAME not in attrs:
                 setattr(cls, META_CLASS_NAME, DefaultMeta)
 
+            if 'DoesNotExist' not in attrs:
```

Comment:
can you add a comment explaining what this code is for? basically repeating what's in the PR description already